frontend/src/components/Comment/Comment.js

```diff
@@ -7,6 +7,7 @@ export const Comment = (props) => {
 
     useEffect(() => {
         setTimePosted(timeElapseSincePosted(new Date(props.comment.created)));
+        //eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can I ask what this does? (or why it was commented?)

Reply:
It disable the warning from react related to dependency. It has to be commented out to disable the warning